rundk/README.md

```diff
@@ -0,0 +1,46 @@
+# rundk
+
+`rundk` is a tool to run a test command from the test image, by using it
+developers can reproduce the test flow as run in the CI environment.
+
+## Prerequisites
+
+- [Docker](https://docs.docker.com/get-docker/) must be installed
+- If you run this tool on MAC, `/tmp` directory must be added to Docker's File
+  Sharing settings to allow Docker accessing the temporary files created for
+  mount, see the [comment](https://github.com/docker/docker.github.io/issues/4709#issuecomment-639596451).
+
+## Installation
+
+`rundk` can be installed and upgraded by running:
+
+```shell
+go get knative.dev/test-infra/rundk
+```
+
+## Usage
+
+```shell
+Usage of rundk:
+  -test-image string
+        The image we use to run the test flow. (default "gcr.io/knative-tests/test-infra/prow-tests:stable")
+  -mounts string
+        A list of extra folders or files separated by comma that need to be mounted to run the test flow.
+  -mandatory-env-vars string
+        A list of env vars separated by comma that must be set on local. (default "GOOGLE_APPLICATION_CREDENTIALS")
+  -optional-env-vars string
+        A list of env vars separated by comma that optionally need to be set on local.
+```
+
+### Example
+
+Run E2E tests for a Knative repository:
+
+```shell
+export GOOGLE_APPLICATION_CREDENTIALS=/temp/gcloud-secret-key.json
+rundk ./test/e2e-tests.sh
```

Comment:
I don't think you should take the script name like this, instead of an argument, because if you run it from a subdirectory and use a relative path, it won't work. It also won't work with relative paths.

Reply:
So maybe take path -> convert to realpath -> confirm under the repo root -> convert to path relative to repo root?

Reply:
It will work, since in this tool we set the working directory as the root directory of the repo, as long as you run this tool in one of the repo's subdirectories.

Reply:
if I'm in the test directory already, and run:
`rundk ./e2e-tests.sh`
it won't work despite auto-completing that way

Reply:
Good point. I have changed the working directory to be the directory where the tool is run.